pkg/debug/scripts/collect-info.sh

```diff
@@ -216,6 +216,37 @@ collect_pillar_backtraces()
 
     echo "- done pillar backtraces"
 }
+collect_zfs_info()
+{
+        type=$(cat /run/eve.persist_type)
+        if [ "$type" = "zfs" ]; then
+           echo "- Collecting ZFS specific info"
+           echo "zpool status" >> "$DIR/zfs-info"
+           echo "============" >> "$DIR/zfs-info"
+           eve exec pillar zpool status >> "$DIR/zfs-info"
```

Comment:
Cool! I didn't know we had such a command to run a command from another container!